src/experimental.service.ts

```diff
@@ -1,14 +1,95 @@
 import { Injectable } from "@angular/core";
 
+/**
+ * Represents options for an experiment. The only required property is `enabled`, however other options may be supplied.
+ */
+export interface Experiment {
+	enabled: boolean;
+	[key: string]: any;
+}
+
 @Injectable()
 export class ExperimentalService {
-	static experimentalEnabled = false;
+	/**
+	 * Tracks if the v10 experimental mode is enabled
+	 * @deprecated
+	 */
+	static experimentalEnabled = true;
 
+	/**
+	 * Sets the v10 experimental mode
+	 * @deprecated
+	 */
 	set isExperimental(v: boolean) {
 		ExperimentalService.experimentalEnabled = v;
+		if (!v) {
+			this.disableExperiment("v10");
```

Comment:
Do we want to keep v10 experimental in v3?

Reply:
Nope